recipes/xarray-extras/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xarray-extras
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xarray-extras/xarray_extras-{{ version }}.tar.gz
+  sha256: b73dc3ee40fdf38c5576383c40a5890d1010e346b72e8cf8f20cf7e8b66f19c8
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python>=3.5
+    - pip
+  run:
+    - python>=3.5
+    - dask>=0.17.3
+    - numba>=0.34
+    - numpy>=1.13
+    - pandas>=0.20
+    - scipy>=1.0
+    - xarray>=0.10
```

Comment:
These need to have a space before the `>=`.

ref: https://github.com/conda/conda-build/issues/2828

Reply:
Funny how your ref uses a mixed style :)
One or the other style should be added to conda-forge-linter...